jax/_src/numpy/util.py

```diff
@@ -92,7 +92,7 @@ def _parse_parameters(body: str) -> Dict[str, str]:
   return OrderedDict((p.partition(' : ')[0].partition(', ')[0], p) for p in parameters)
 
 
-def _wraps(fun: Callable, update_doc: bool = True, lax_description: str = "",
+def _wraps(fun: Optional[Callable], update_doc: bool = True, lax_description: str = "",
```

Comment:
Why is this optional now?

Reply:
We treat it as optional here: https://github.com/google/jax/blob/e3fe4a2c7c31ce56b6b99469b12a05e43a57727c/jax/_src/numpy/lax_numpy.py#L2207